src/platforms/hosted/windows/ftdi.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2011  Black Sphere Technologies Ltd.
+ * Copyright (C) 2018  Uwe Bonnes(bon@elektron.ikp.physik.tu-darmstadt.de)
+ * Written by Gareth McMullin <gareth@blacksphere.co.nz>
```

Comment:
As this and the header are brand new files, you can drop these 3 lines and license this however you choose (GPLv3+ is not a hard requirement - you could choose BSD-3-Clause for example)